cetmix_tower_server/models/cx_tower_variable_value.py

```diff
@@ -241,6 +240,22 @@ def _inverse_value_char(self):
                         )
                     )
 
+    # Workaround for the default value not being set
+    @api.model_create_multi
+    def create(self, vals_list):
+        variable_obj = self.env["cx.tower.variable"]
+        for vals in vals_list:
+            # Set access level from the variable
+            # if not provided explicitly
+            access_level = vals.get("access_level")
+            if access_level:
+                continue
+            variable_id = vals.get("variable_id")
+            if variable_id:
+                variable = variable_obj.browse(variable_id)
+                vals["access_level"] = variable.access_level
```

Comment:
can you use compute method?

Reply:
> can you use compute method?

No, this is not possible because there is a 'default' key set at the mixin level. So compute method is not triggered on record creation, only when a record will be updated next time.
In this case if we replace a default value with `None` it will try to create a new record with `access level = None` which will either raise an Exception if this field is required, or will require access rules modifications.
So using `create` is the only reasonable workaround for that.